ci-operator/config/openshift/external-dns-operator/openshift-external-dns-operator-release-1.1.yaml

```diff
@@ -0,0 +1,152 @@
+    namespace: ocp
+    tag: kube-rbac-proxy
+  ocp_builder_rhel-8-golang-1.19-openshift-4.12:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.19-openshift-4.12
+  operator-index:
+    name: redhat-operator-index
+    namespace: ci
+    tag: v4.9
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: Dockerfile.openshift
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.19-openshift-4.12:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.12
+  to: external-dns-operator
+operator:
+  bundles:
+  - as: external-dns-operator-bundle
+    base_index: operator-index
+    dockerfile_path: Dockerfile.bundle
+  substitutions:
+  - pullspec: quay.io/openshift/origin-external-dns-operator:latest
+    with: pipeline:external-dns-operator
+  - pullspec: quay.io/openshift/origin-kube-rbac-proxy:latest
+    with: pipeline:kube-rbac-proxy
+releases:
+  initial:
+    integration:
+      name: "4.12"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.12"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: verify
+  commands: |
+    export GOLANGCI_LINT_CACHE=$(mktemp -d)
+    make verify
+  container:
+    from: src
+- as: unit
+  commands: |
+    export XDG_CACHE_HOME=$(mktemp -d)
+    make test
+  container:
+    from: src
+- as: e2e-aws-operator
+  steps:
+    cluster_profile: aws
+    dependencies:
+      OO_INDEX: ci-index-external-dns-operator-bundle
+    env:
+      OO_CHANNEL: stable-v1.1
+      OO_INSTALL_NAMESPACE: external-dns-operator
+      OO_PACKAGE: external-dns-operator
+      OO_TARGET_NAMESPACES: '!install'
+    test:
+    - as: test
+      cli: latest
+      commands: make test-e2e
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: optional-operators-ci-aws
+- as: e2e-gcp-operator
+  steps:
+    cluster_profile: gcp
+    dependencies:
+      OO_INDEX: ci-index-external-dns-operator-bundle
+    env:
+      OO_CHANNEL: stable-v1.1
+      OO_INSTALL_NAMESPACE: external-dns-operator
+      OO_PACKAGE: external-dns-operator
+      OO_TARGET_NAMESPACES: '!install'
+    test:
+    - as: test
+      cli: latest
+      commands: make test-e2e
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: optional-operators-ci-gcp
+- as: e2e-azure-operator
+  steps:
+    cluster_profile: azure4
+    dependencies:
+      OO_INDEX: ci-index-external-dns-operator-bundle
+    env:
+      OO_CHANNEL: stable-v1.1
+      OO_INSTALL_NAMESPACE: external-dns-operator
+      OO_PACKAGE: external-dns-operator
+      OO_TARGET_NAMESPACES: '!install'
+    test:
+    - as: test
+      cli: latest
+      commands: make test-e2e
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: optional-operators-ci-azure
+- as: e2e-infoblox-operator
+  steps:
+    cluster_profile: azure4
```

Comment:
infoblox uses the same cluster_profile as azure?

Reply:
Yes. It doesn't really matter which cloud provider the Infoblox test case will be run at. Since we don't request the credentials from the CloudCredentialsOperator of the cluster like for AWS, GCP and Azure. So, the choice of the cluster profile is pure convenience for Infoblox case - we chose Azure recently due to a lot of issues on GCP and AWS CI accounts.

Reply:
I noticed the failure of `e2e-azure-ovn-infoblox-operator` test.  Just wondering if something needs to change here.

Reply:
No, the temporary Infloblox licenses expired, they last 3 months. We still use the temporary licenses for Infoblox product from AWS marketplace as we didn't manage to get any from Infoblox (company). We'll have to re-iterate with DPTP on this.